nest-forms-backend/src/app.module.ts

```diff
@@ -58,16 +61,16 @@ import SharepointSubservice from './utils/subservices/sharepoint.subservice'
     // BEWARE: If Bull doesn't connect to Redis successfully, it will silently fail!
     // https://github.com/nestjs/bull/issues/1076
     BullModule.forRootAsync({
-      imports: [ConfigModule],
-      useFactory: async (configService: ConfigService) => ({
+      imports: [BaConfigModule],
+      inject: [BaConfigService],
+      useFactory: async (baConfigService: BaConfigService) => ({
         redis: {
-          host: <string>configService.get('REDIS_SERVICE'),
-          port: parseInt(<string>configService.get('REDIS_PORT') ?? '6379', 10),
```

Comment:
This default value 6379 is now lost. Maybe it was not desirable in the first place, but it was there.

Reply:
Yes, but that is one of the selling points of validated config (we can be sure the value is defined and we can remove these impromptu defaults).

The defaults that we can find in code are mostly there to satisfy type checker, because otherwise it will fail (in this case, you cannot pass `undefined` to `parseInt`, so someone decided to provide a default value. In many cases there are not really meaningful (empty strings).